test/suites/integration/suite_test.go

```diff
@@ -3,11 +3,11 @@ package integration
 import (
 	"context"
 	"os"
-	"testing"
 
 	"github.com/aws/aws-application-networking-k8s/test/pkg/test"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"testing"
```

Comment:
interesting that formater you are using mixing up imports order, usually it's std lib, then public packages, then local packages, you might need to adjust IDE setting for go formatting